api-reference/10 UI Widgets/GridBase/1 Configuration/rowDragging/allowDropInsideItem.md

```diff
@@ -8,8 +8,9 @@ default: false
 Allows users to drop a row inside another row.
 
 ---
-This option allows users to drop rows only in the UI. You should also handle drag and drop in code. If users drag and drop rows inside a single component, use the [onReorder]({basewidgetpath}/Configuration/rowDragging/#onReorder) handler. To allow users to drag and drop rows between components, use the [onAdd]({basewidgetpath}/Configuration/rowDragging/#onAdd) handler.
+This option allows users to drop rows only in the UI. You should also handle drag and drop in code. Implement the [onReorder]({basewidgetpath}/Configuration/rowDragging/#onReorder) handler to allow users to drop rows inside a single component. 
 
-#include common-demobutton with {
-    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/LocalReordering"
-} 
+Refer to the [group]({basewidgetpath}/Configuration/rowDragging/#group) topic to see how to configure cross-component drag and drop.
```

Comment:
> Refer **to** the group topic **to** see how **to** configure.

- Too many to's.
- The group topic doesn't exactly _show_ the configuration, it only explains how to configure.